### PR TITLE
tiflash: remove tiflash tar

### PR DESCRIPTION
**Pull request by marsishandsome**
### What is changed, added or deleted? (Required)
remove tiflash tar

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/tiflash/deploy.md [hunk 1]
```diff
@@ -54,19 +54,24 @@ TiFlash 支持多目录存储，所以无需使用 RAID。
 
 目前对于全新部署 TiFlash 场景，推荐通过下载离线安装包来部署 TiFlash。步骤如下：
 
-1. 下载对应版本的离线包，并解压：
+1. 下载 TiDB 3.1 版本对应 tag 的 tidb-ansible
```
Comment (yikeke) [inline reference/tiflash/deploy.md:57]:
```suggestion
1. 下载 TiDB 3.1 版本对应 tag 的 TiDB Ansible：
```

File: reference/tiflash/deploy.md [hunk 2]
```diff
@@ -54,19 +54,24 @@ TiFlash 支持多目录存储，所以无需使用 RAID。
...
 
-    - 如果你使用的是 TiDB 3.1 rc 版，执行以下命令：
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        git clone -b $tag https://github.com/pingcap/tidb-ansible.git
+        ```
+
+2. 下载 binary
```
Comment (yikeke) [inline reference/tiflash/deploy.md:65]:
```suggestion
2. 下载 TiDB Ansible 的二进制文件：
```
下载什么的二进制文件？TiDB Ansible 的？

Comment (marsishandsome) [reply reference/tiflash/deploy.md:65]: `local prepare`命令会下载tidb tikv pd等组件的二进制安装文件

Comment (yikeke) [reply reference/tiflash/deploy.md:65]: 调整了下 suggestion，这样可以吗

**Pull request merged**